src/containers/Questions/questions.jsx

```diff
@@ -0,0 +1,215 @@
+  };
+
+  skip = () => {
+    const { current } = this.state;
+    if (current < entryData.length - 1) {
+      const curr = current + 1;
+      this.setState({ current: curr, errors: {} });
+    } else {
+      const { history } = this.props;
+      if (journals.length !== 0) {
+        message.success('Yes, you have added a journal');
+        history.push('/home');
+        // here, a request will be post to firebase to save data
+        // that will be as follows : [{title:'', content:'', time:'', date:'',month:''}]
+      }
+      message.warning("You didn't make an entry today");
+      history.push('/home');
+    }
+  };
+
+  render() {
+    const { errors, current, title, content } = this.state;
+    const {
+      history: { goBack },
+    } = this.props;
+    return (
+      <div>
+        <div className="question__navigation">
+          {current > 0 ? (
+            <Icon type="left" onClick={() => this.prev()} />
+          ) : (
+            <BackButton handleBack={goBack} />
+          )}
+          <div>
+            <Popconfirm
+              title="Do you really want to exit?"
+              onConfirm={this.confirm}
+              onCancel={this.cancel}
+              okText="Yes"
+              cancelText="cancel"
+            >
+              <Icon type="close" />
+            </Popconfirm>
+          </div>
+        </div>
+        <div className="question__type">
+          <p>
+            Question 3·/
+            {current + 1}
+          </p>
+          <Progress
+            percent={entryData[current].percent}
+            size="small"
+            showInfo={false}
+          />
+          <p>{entryData[current].heading}</p>
+          <div className="question__title">
+            <Input
+              placeholder="Title"
+              name="title"
+              value={title}
+              autoComplete="off"
+              onChange={this.handleChange}
+            />
+            {errors.title && (
+              <span className="question__error-field">{errors.title}</span>
+            )}
+          </div>
+          <div className="question__content">
+            <TextArea
+              rows={4}
+              placeholder="Write your words"
+              name="content"
+              value={content}
+              onChange={this.handleChange}
+            />
+            {errors.content && (
+              <span className="question__error-field">{errors.content}</span>
+            )}
+          </div>
+        </div>
+
+        <div className="question__steps-action">
+          {current < entryData.length - 1 && (
+            <Button type="primary" onClick={() => this.next()}>
+              Next
+            </Button>
+          )}
+          {current === entryData.length - 1 && (
+            <Button
+              type="primary"
+              onClick={() => {
+                this.finish();
+              }}
+            >
+              Done
+            </Button>
+          )}
```

Comment:
you can mix the condition for these two buttons  because we need to appear just one of both so look to this code 
```jsx
        {current < entryData.length - 1 ? (
          <Button type="primary" onClick={() => handleNext()}>
            Next
          </Button>
        ) : (
          <Button type="primary" onClick={() => handleSubmit()}>
            Done
          </Button>
        )}
```